src/locales/zh-CN.po

```diff
@@ -1677,14 +1677,14 @@ msgstr ""
 
 #: src/components/swapv2/TradeTypeSelection.tsx
 msgid "Maximum Return"
-msgstr "最大的收益"
+msgstr ""
```

Comment:
Why do you remove this translate

Reply:
It is generated when I install packages.

Reply:
Revert these translation files. Then run yarn i18n again.

Reply:
Done.